src/main/resources/templates/sql-queries

```diff
@@ -1,6 +1,6 @@
 
-CREATE TABLE car (
-					id_car bigint not null auto_increment primary key,
+CREATE TABLE cars (
```

Comment:
SQL files should be under the resources but in a dedicated package. Usually, templates are used for Thymeleaf or something similar.